src/main/java/console/web3j/Web3jImpl.java

```diff
@@ -445,6 +445,7 @@ public void getTransactionReceipt(String[] params) throws Exception {
                 }
             }
             if (!Common.EMPTY_OUTPUT.equals(receipt.getOutput())) {
+                TxDecodeUtil.setInputForReceipt(web3j, receipt);
                 TxDecodeUtil.decodeOutput(abi, receipt);
             }
             if (receipt.getLogs().size() != 0) {
```

Comment:
if receipt.getLogs().size() != null is needed?

Reply:
added check null